--- a/src/main.cpp
+++ b/src/main.cpp
@@ -28,8 +28,6 @@
 int main(int argc, char* argv[])
 {
      QApplication* app = new QApplication(argc,argv);
-     QTextCodec::setCodecForTr(QTextCodec::codecForName("UTF-8"));
-     QTextCodec::setCodecForCStrings(QTextCodec::codecForName("UTF-8"));
      QString pro = "";
      if(argc == 2 && QFile::exists(QString(argv[1])))
 	  pro = argv[1];

--- a/src/optimizerdialog.cpp
+++ b/src/optimizerdialog.cpp
@@ -32,9 +32,9 @@
      showBox->setChecked(set->value("show_optimizer",false).toBool());
      proc = new QProcess(this);
     
-     if(set->value("convert_exec","").toString().isEmpty())
+/*     if(set->value("convert_exec","").toString().isEmpty())
 	  set->setValue("convert_exec",findConvert());
-     checkIM();
+     checkIM();*/
 
      connect(closeButton, SIGNAL(clicked()), this, SLOT(close()));
      connect(srcButton, SIGNAL(clicked()), this, SLOT(setSrc()));
@@ -179,7 +179,7 @@
      return s.split(";",QString::SkipEmptyParts);
 #endif
 }
-
+/*
 QString OptimizerDialog::findConvert()
 {
      QString exec =
@@ -197,4 +197,4 @@
 	       return env.at(i)+exec;
 	  }
      return "";
-}
+}*/

--- a/src/optimizerdialog.h
+++ b/src/optimizerdialog.h
@@ -37,7 +37,7 @@
      bool convertAvailable();
      void checkIM();
      QStringList sysEnv();
-     QString findConvert();
+//     QString findConvert();
      QProcess* proc;
      QSettings* set;
      private slots:
