--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -3379,7 +3379,6 @@
 HKLM,%CurrentVersionNT%\SvcHost,"netsvcs",0x00010008,"Schedule"
 
 [Services]
-HKLM,%CurrentVersion%\RunServices,"winemenubuilder",2,"%11%\winemenubuilder.exe -a -r"
 HKLM,"System\CurrentControlSet\Services\Eventlog\Application",,16
 HKLM,"System\CurrentControlSet\Services\Eventlog\System",,16
 HKLM,"System\CurrentControlSet\Services\Tcpip\Parameters",,16
