--- a/src/forms/frmoptions.cpp
+++ b/src/forms/frmoptions.cpp
@@ -435,8 +435,8 @@
 {
     GlobalConfig().setP7zipPath("");
     GlobalConfig().setTmpPath(QDir::tempPath());
-    GlobalConfig().setLanguage("pl");
-    GlobalConfig().setLanguageBackup("en");
+    GlobalConfig().setLanguage("en");
+    GlobalConfig().setLanguageBackup("de");
     GlobalConfig().setNoBackup(false);
     GlobalConfig().setQuietBatch(false);
     GlobalConfig().setChangePermissions(false);
