--- a/doc/qnapi-download.desktop
+++ b/doc/qnapi-download.desktop
@@ -1,14 +1,14 @@
 [Desktop Entry]
 Version=1.0
 Actions=QNapiDownload;
-Name=Pobierz napisy do filmu z QNapi
-GenericName=Pobierz napisy do filmu z QNapi
+Name=Download subtitles with QNapi
+GenericName=Download subtitles with QNapi
 Icon=qnapi-48
 ServiceTypes=video/x-msvideo,video/x-ms-asf,video/mpeg,video/x-ms-wmv,video/mp4,video/quicktime,video/x-theora,video/x-matroska,video/3gpp,application/vnd.rn-realmedia,application/vnd.rn-realmedia-vbr
 X-KDE-ServiceTypes=KonqPopupMenu/Plugin
 Type=Service
 
 [Desktop Action QNapiDownload]
-Name=Pobierz napisy do filmu z QNapi
+Name=Download subtitles with QNapi
 Exec=qnapi %F
 Icon=qnapi-48

--- a/doc/qnapi-scan.desktop
+++ b/doc/qnapi-scan.desktop
@@ -1,14 +1,14 @@
 [Desktop Entry]
 Version=1.0
 Actions=QNapiScan;
-Name=Przeskanuj katalog i pobierz napisy
-GenericName=Przeskanuj katalogu i pobierz napisy
+Name=Scan directory and download subtitles
+GenericName=Scan directory and download subtitles
 Icon=qnapi-48
 ServiceTypes=inode/directory
 X-KDE-ServiceTypes=KonqPopupMenu/Plugin
 Type=Service
 
 [Desktop Action QNapiScan]
-Name=Przeskanuj katalog i pobierz napisy
+Name=Scan directory and download subtitles
 Exec=qnapi %U
 Icon=qnapi-48

--- a/doc/qnapi.desktop
+++ b/doc/qnapi.desktop
@@ -4,7 +4,7 @@
 Icon=qnapi
 Type=Application
 Categories=AudioVideo;
-Comment=Program do pobierania napisów do filmów
-GenericName=Program do pobierania napisów do filmów
+Comment=Program for downloading video subtitles
+GenericName=Program for downloading video subtitles
 Terminal=false
 X-KDE-StartupNotify=false
