Description: add a necessary header. (Closes: #518851)
Author: peter green <plugwash@p10link.net>
Last-Update: 2009-04-05
Index: sockstat-0.3/sockstat.c
===================================================================
--- a/sockstat.c	2009-06-04 11:34:59.051534157 +0800
+++ b/sockstat.c	2009-06-04 11:35:27.283531833 +0800
@@ -41,6 +41,7 @@
 #include <pwd.h>
 #include <grp.h>
 #include <assert.h>
+#include <limits.h>
 
 #define SEARCH_ALL     0x00
 #define SEARCH_GID     0x01
