Description: add GCC hardening.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2016-12-30
Index: sockstat-0.3/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@ SRCS = sockstat.c
 OBJS = ${SRCS:.c=.o}
 
 sockstat: ${OBJS}
-	cc -o $@ ${OBJS}
+	cc -o $@ $(LDFLAGS) ${OBJS}
 
 install:
 	install -m 755 -c sockstat ${DESTDIR}/usr/bin
