--- a/dreamdesktop.cpp
+++ b/dreamdesktop.cpp
@@ -256,7 +256,7 @@
                     case DD_F_SPLINE: useFilter = SWS_SPLINE; break;
                 }
 
-                SwsContext *img_convert_ctx = sws_getContext(pCodecCtx->width, pCodecCtx->height, pCodecCtx->pix_fmt, dd->boundingWidth, dd->boundingHeight, PIX_FMT_RGB32, useFilter, NULL, NULL, NULL);
+                SwsContext *img_convert_ctx = sws_getContext(pCodecCtx->width, pCodecCtx->height, pCodecCtx->pix_fmt, pCodecCtx->width, pCodecCtx->height, PIX_FMT_RGB32, useFilter, NULL, NULL, NULL);
 
                 dd->mutex->lock();
 
